lib/src/tag_detection.dart

```diff
@@ -495,6 +533,54 @@ class Sdk {
   static List<Sdk> knownSdks = [dart, flutter];
 }
 
+/// A package is said to null-safety compliant if:
+///
+/// The package has opted-in by specifying a lower dart sdk bound >= 2.10
+/// No libraries in the package opts out of the null-safety enabled language
+/// version.
+/// All (non-dev) dependencies in the latest version resolvable by pub are
+/// fully null-safety-compliant by this definition.
```

Comment:
Maybe a little markdown can make this prettier :D

Reply:
Sure - done :)